Sources/PinLayout.swift

```diff
@@ -83,6 +83,14 @@ public class PinLayout<View: Layoutable> {
         }
         apply()
     }
+    
+    #if os(iOS)
+    public var keyboardLayout: PEdgeInsets {
```

Comment:
After more thinking, it would better match other properties if we would name it `keyboardMargins`

* `UIView.readableContentGuide` is named `pin.readableMargins`
* `UIView.layoutMarginsGuide` is named `pin.layoutMargins`

Then I would propose
* `UIView.keyboardLayoutGuide` named `pin.keyboardMargins`

What do you think?

Reply:
Good idea.